great-idea/css/index.css

```diff
@@ -63,4 +63,283 @@ h1, h2, h3, h4, h5 {
     margin-bottom: 15px;
 }
 
-/* Copy and paste your work from yesterday here and start to refactor into flexbox */
\ No newline at end of file
+/* Copy and paste your work from yesterday here and start to refactor into flexbox */
+
+ 
+
+*{
+	
+	box-sizing: border-box;
+	padding:0;
+	margin:0;
+	max-width:100%;
+	
+}
+
+
+body{
+
+	margin: 0 auto;
+	width: 1000px
+}
+@media screen and (min-width: 849px){
```

Comment:
Nice! You're ready to tackle upcoming media query modules. Great that you got familiar with them for Stretch